server/reviews.js

```diff
@@ -0,0 +1,22 @@
+const axios = require('axios');
+require('dotenv').config(); // allow server to read .env for environmental variables
+
+const reviewUrl = 'https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/reviews';
+const apiKey = process.env.API_KEY;
+
+const getReviews = (id) => {
+  return axios.get(reviewUrl, {
+  headers: { Authorization: apiKey },
+  params: { product_id: id },
+})};
```

Comment:
I like the use of the params key in the options object of your get request, better readability. The way I set the url in my axios functions looks pretty funky, I think I'll edit to use params too

Reply:
lol I kinda have to use this as it's specified in Glearn, there doesn't seem to be a path for `reviews/:product_id`, so I think what you did with getting products is actually the right way to go? (but you can definitely try it out to see whether it works!)